### PR TITLE
[FIX] account_payment_order_to_voucher: Bad field name (vouchers_ids) and filter in view

### PR DESCRIPTION
**Pull request by ozono**
account_payment.view_payment_order_form not loading due to error in inherited view: bad field name (vouchers_ids) and filter in view_payment_order_form  :sweat_smile:

### CODE REVIEW

File: account_payment_order_to_voucher/views/payment_order_view.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
                 <div class=" oe_right oe_button_box" position="after">
                     <div class=" oe_right oe_button_box">
                         <button class="oe_inline oe_stat_button oe_right" name="generate_vouchers" string="Create vouchers"
-                        type="object" attrs="{'invisible':['|',('state','!=','done'),('vouchers_ids','!=',False)]}"
+                        type="object" attrs="{'invisible':['|',('state','!=','done'),('voucher_ids','!=',[(6, False, [])])]}"
```
Comment (eLBati) [inline account_payment_order_to_voucher/views/payment_order_view.xml]: I am not sure about `[(6, False, [])]`. Do you have any example where it is already used?

**Comment by ozono** [comment]
Reasons are explained here http://ludwiktrammer.github.io/odoo/domain-for-empty-many2many-in-attrs.html

File: account_payment_order_to_voucher/views/payment_order_view.xml (hunk 1 above)
Comment (ozono) [inline account_payment_order_to_voucher/views/payment_order_view.xml]: Reasons are explained here http://ludwiktrammer.github.io/odoo/domain-for-empty-many2many-in-attrs.html

Comment (eLBati) [inline account_payment_order_to_voucher/views/payment_order_view.xml]: Thanks

**Comment by eLBati** [comment]
:+1:

**Comment by ozono** [comment]
Merged with branch '8.0' of https://github.com/OCA/account-payment with a more recent PR (https://github.com/OCA/account-payment/pull/99 that includes one fix this PR already was doing.

**Comment by eLBati** [comment]
Thanks @ozono could you rebase on top of OCA:8.0 in order to only add 1 commit?

**Pull request closed without merging**